src/tools/gcc.lua

```diff
@@ -684,5 +684,5 @@
 		if ((cfg.gccprefix  or version ~= "") and gcc.tools[tool]) or tool == "rc" then
 			return (cfg.gccprefix or "") .. gcc.tools[tool] .. version
 		end
-		return nil
+		return gcc.tools[tool]
```

Comment:
Previous condition might be removed completely and just return
`return (cfg.gccprefix or "") .. gcc.tools[tool] .. version`